looker_deployer/commands/deploy_content.py

```diff
@@ -19,28 +19,36 @@
 import tempfile
 import shutil
 import threading
+import json
 from concurrent.futures import ThreadPoolExecutor
 from itertools import repeat
 from looker_deployer.utils import deploy_logging
 from looker_deployer.utils import parse_ini
 from looker_deployer.utils.get_client import get_client
-from looker_sdk import models
+from looker_sdk import models40 as models
 
 
 logger = deploy_logging.get_logger(__name__)
 
+def alert_cleanup(sdk):
+    logger.debug("cleaning up orphaned alerts")
+    disabled_alerts = sdk.search_alerts(disabled="true")
+    for alert in disabled_alerts:
+        if alert['disabled_reason'] == "Dashboard element has been removed.":
+            sdk.delete_alert(alert['id'])
+
 
 def get_space_ids_from_name(space_name, parent_id, sdk):
```

Comment:
Can we update any functions from space to folder and variables? Also tests will need to be updated for this change. Just don't change the actual looker commands (which already referred to things as folders). I'm pretty sure space to folder find & replace should fix 95% of use cases.